src/JDBC.jl

```diff
@@ -548,7 +571,7 @@ global const get_method_dict = Dict(
         # JDBC_COLTYPE_CLOB => 2005,
         # JDBC_COLTYPE_DATALINK => 70,
         JDBC_COLTYPE_DATE => getDate,
-        JDBC_COLTYPE_DECIMAL => getFloat,
+        JDBC_COLTYPE_DECIMAL => getBigDecimal,
```

Comment:
So one of the reasons I hadn't implemented this yet is the fear that doing things this way means every numeric field is now Decimal object, which is hundreds of time slower. Even columsn with small or low precision numbers. Should I not worry about that?

Reply:
I was also concerned about just what the meaning was of `NUMERIC`.  As far as I can tell, `NUMERIC` really does mean decimal, and not float, see [here](https://docs.oracle.com/javase/1.5.0/docs/guide/jdbc/getstart/mapping.html).  Even if this is true, it doesn't mean there aren't lots of places where people have carelessly declared fields that should be floats as `NUMERIC`, but not much we can do about that I suppose.